content/en/_index.md

```diff
@@ -11,10 +11,7 @@ show_banner: true
 <div class="l-primary-buttons mt-5">
 
 - [Learn more](/docs/concepts/)
-- [Mission and vision][]
-
-[mission and vision]:
-  https://github.com/open-telemetry/community/blob/main/mission-vision-values.md#otel-mission-vision-and-values
+- [Mission, Vision & Values](/mission-vision-and-values)
```

Comment:
- Change button text to sentence case
- Use `/mission` as the path (which implies renaming the file, as is noted below)

Reply:
fixed in ab513c6